src/AppContainer.tsx

```diff
@@ -61,7 +62,10 @@ export const AppContainer = ({
                 <LayoutStateProvider>
                   <WhiteboardHotkeysProvider>
                     <GuidedTourProvider>
-                      <App />
+                      <SnackbarProvider>
+                        <Snackbar />
```

Comment:
Should we move `Snackbar` into a place where app is initialized/loaded already? For example where `GuidedTour` is located: Layout.tsx:70